src/wui/interactive_base.cc

```diff
@@ -1452,9 +1453,15 @@ bool InteractiveBase::handle_key(bool const down, SDL_Keysym const code) {
 			   this, debugconsole_, *DebugConsole::get_chat_provider());
 			return true;
 		case SDLK_F3:
-			if (cheat_mode_enabled_ || (code.mod & KMOD_CTRL)) {
-				cheat_mode_enabled_ = !cheat_mode_enabled_;
-				return true;
+			if (cheat_mode_enabled_) {
+				cheat_mode_enabled_ = false;
+			} else if (code.mod & KMOD_CTRL) {
+				if (chat_provider_) {
+					/** TRANSLATORS: This is a chat message which is automatically sent */
```

Comment:
/** TRANSLATORS: This is a chat message which is automatically sent to all players when a player enables cheating mode */